### DISCUSSION TITLE
Support for map tiles in the 3D panel

### DISCUSSION THREAD

**Discussion by pickledgator** [foxglove/community#808] [Q&A]
It would be amazing if there was a way for the 3D panel and the map panel to be combined such that you were able to render both custom map tiles and dense geometric primitives on the same visualization panel.

**Comment by jtbandes** [comment]
Related: #303, #319

**Comment by cshang412** [comment]
i asked the same question on[ slack](https://foxglove.slack.com/archives/C028UEY858S/p1692949546253729) , To promote the development of this feature i will add here too.
this feature is pretty important i think.


![图片](https://github.com/foxglove/community/assets/29396384/d736b9c6-1aad-4071-9d42-185990c404c0)
![图片](https://github.com/foxglove/community/assets/29396384/233d2c73-d0a6-402e-b448-d8c3ea64bf53)
